assets/Styles/main.css

```diff
@@ -0,0 +1,904 @@
+
+  #socialBar .social-icons {
+    width: 100%;
+    padding-right: 3rem;
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: flex-end;
+  }
+
+  #partContainer img {
+    width: 20%;
+  }
+
+  #aboutLogo img {
+    width: 40%;
+    height: auto;
+    padding: 3rem 4rem;
+    margin-top: 2rem;
+  }
+
+  #pastEvents img {
+    margin: 0 1rem;
+  }
+
+  #socialBar .social-icons img {
+    margin: 0 1rem;
+    width: auto;
+    height: 25%;
+    cursor: pointer;
+  }
+
+  #socialBar .social-icons a {
+    margin: 0 1rem;
+    text-decoration: none;
+    color: var(--primary-white);
+  }
+
+  #heroSection {
+    width: 100%;
+    align-items: center;
+    padding-bottom: 7rem;
+    background-image:
+      url("/assets/png/Ellipse-color-pattern.png"),
+      url("/assets/png/deco-0-icon.png"),
+      url("/assets/png/deco-yen-icon.png"),
+      url("/assets/png/deco-0-icon.png"),
+      url("/assets/png/square-B&W.png"),
+      url("/assets/png/deco-c-icon.png");
+    background-repeat: no-repeat;
+    background-size: 9%, 2%, auto, 2%, 6%, 3%;
+    background-position: 60% 70%, 13% 41%, 5% 98%, 99% 47%, 93% 72%, 75% 93%;
+  }
+
+  #backgroundDesk {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 100vw;
+    height: 100%;
+    padding-top: 9rem;
+    background-image: url("/assets/svg/desk-header.svg");
+    background-size: contain;
+    background-repeat: no-repeat;
+    margin-top: -2%;
+  }
+
+  #backgroundDesk .hero-content {
+    width: 75%;
+    padding-left: 20%;
+  }
+
+  #backgroundDesk .hero-content h3 {
+    font-size: 45px;
+    align-self: flex-start;
+  }
+
+  #backgroundDesk .hero-content h2 {
+    align-self: flex-start;
+    font-size: 40px;
+  }
+
+  #sumContainer {
+    width: 70%;
+    border: 2px solid white;
+    font-size: 14px;
+  }
+
+  #actContainer {
+    display: grid;
+    grid-template-columns: 20% 20% 20% 20% 20%;
+    padding: 0 10rem 4rem 10rem;
+  }
+
+  #actContainer li {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    text-align: center;
+    justify-items: center;
+  }
+
+  #featSpeakers {
+    padding: 2rem 4rem;
+  }
+
+  #featContainer {
+    flex-direction: row;
+    flex-wrap: wrap;
+    gap: 10px;
+  }
+
+  .featCard {
+    width: 40vw;
+  }
+
+  .featCard h4 {
+    font-size: 19px;
+  }
+
+  .featCard h5 {
+    font-size: 16px;
+    padding-bottom: 5px;
+  }
+
+  footer p {
+    font-size: 18px;
+  }
+
+  .featCard p {
+    font-size: 15px;
+  }
+
+  #partners {
+    padding: 2rem 5rem;
+  }
+
+  #partContainer {
+    padding: 1rem 2rem;
+    justify-content: center;
+  }
+
+  #partnersAbt {
+    display: none;
+  }
+
+  footer {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+  }
+
+  footer div {
+    width: 25%;
+    padding: 3rem 3rem 2rem 2rem;
+  }
+
+  #menuMobile-abt {
+    display: none;
+  }
+
+  #aboutMain {
+    padding-top: 7rem;
+    background-image:
```

Comment:
- [ ] **These background images are not loaded in the live version of your website. Please fix it**

The images are not loaded probably because of the bad path. Try to use a relative path to see if it fixes the issue.

Reply:
Fixed :heavy_check_mark: